src/components/footer/index.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link';
+
+export default function Footer() {
+  return (
+    <div className="w-full p-5 flex justify-between items-end bg-black text-white bg-footer-wave bg-cover font-sans">
```

Comment:
This footer seems to be remaining at the top and not at the bottom of the screen. Also you should definitely use <footer> tag, not a div.